server/src/controllers/user.js

```diff
@@ -0,0 +1,95 @@
+const { body, validationResult } = require('express-validator');
+const User = require("../models/userModel");
+const bcrypt = require("bcryptjs");
+
+exports.validateRegister =
+    [
+        body('email')
+            .exists()
```

Comment:
all of these validation requirements are just suggestions as of now. If we do go for these requirements, we should show these requirements on the front end as well